.github/workflows/ping-other-repos.yml

```diff
@@ -0,0 +1,25 @@
+name: Remote Dispatch Action Initiator
+
+on:
+  push:
+  repository_dispatch:
+
+jobs:
+  ping-pong:
+    runs-on: ubuntu-latest
```

Comment:
As an alternative (or in addition) to guarding against unset secrets, we could add an `if` here to prevent running the workflow on forks:

(I think this is right—I haven't tested it.)

```suggestion
    runs-on: ubuntu-latest
    if: ${{ github.respository == 'mdn/browser-compat-data' }}
```